.github/workflows/codespell.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: "Codespell"
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}-${{ matrix.language }}-${{ github.ref }}
```

Comment:
Looks like `${{ matrix.language }}` came from the codeql workflow and can be removed.